src/_test_/Home.test.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import Home from '../components/Home';
+
+describe('Home', () => {
+  test('should render the component', () => {
+    const component = renderer.create(<Home />);
+    const tree = component.toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+});
```

Comment:
- [ ] Well done making most of the tests passed 👏  , however there are still some you still need to fix . I know they may be working on your computer but always remember that most of the product you will make will be running on clients computers 🆗  , Kindly fix the unpassed tests to meet the clients need 😄 😄 😄 
- [ ] Also I noticed that there is no snapshot test running , kindly fix that also 👍 

![Capture](https://github.com/SAZZAD1Q2/math_magician/assets/80458199/14ce1b5a-7a2a-470c-a141-faa68c3de1f0)

Reply:
Dear @iambenkis,
Thank you very much for reviewing my pr. I appreciate your informative advice. The test for the project is working well on my computer. Will you suggest to me how to test for all computers? Besides I used the snapShots test for my project.
Thank you once again.
Md Sazzad Hossain